t/admin/delete-plugin-configs.t

```diff
@@ -0,0 +1,174 @@
+#
```

Comment:
Could we add it to `t/admin/plugin-configs.t`?

Reply:
We should not add more new `delete-*` files. For example, the new case in https://github.com/apache/apisix/pull/4575 is added in `proto3.t`.
I also submit a PR to remove the stale cases: https://github.com/apache/apisix/pull/6121

Reply:
Got it.